changelogs/fragments/1681-add_passwordstore_yaml_support.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - passwordstore lookup plugin - now parsing a password store entry as YAML if possible, skipping the first line (which by convention only contains the password and nothing else). If it cannot be parsed as YAML, the old ``key: value`` parser will be used to process the entry (https://github.com/ansible-collections/community.general/issues/1673)
```

We might need the quotes so this doesn't parse as a YAML dictionary... (And the period at the end is required as well.)
```suggestion
  - "passwordstore lookup plugin - now parsing a password store entry as YAML if possible, skipping the first line (which by convention only contains the password and nothing else). If it cannot be parsed as YAML, the old ``key: value`` parser will be used to process the entry (https://github.com/ansible-collections/community.general/issues/1673)."
```